src/Core/ISolutionWorkspaceService.cs

```diff
@@ -33,6 +33,6 @@ public interface ISolutionWorkspaceService
         /// List Files in a solution
         /// </summary>
         /// <returns>Returns empty if it's not in solution Mode</returns>
-        IEnumerable<string> ListFiles();
+        ICollection<string> ListFiles();
```

Comment:
Sam eas IFolderWorkspaceSErver: Probably an array

Reply:
Changed to IReadOnlyCollection